nautobot/docs/configuration/optional-settings.md

```diff
@@ -234,6 +222,19 @@ EXEMPT_VIEW_PERMISSIONS = ['*']
 
 ---
 
+## FORCE_SCRIPT_NAME
+
+Default: `None`
+
+If not `None`, this will be used as the value of the `SCRIPT_NAME` environment variable in any HTTP request. This setting can be used to override the server-provided value of `SCRIPT_NAME`, which is most commonly used for hosting Nautobot in a subdirectory (e.g. *example.com/nautobot*).
+
+!!! important
+    To host Nautobot under a subdirectory you must set this value to match the same prefix configured on your HTTP server. For example, if you configure NGINX to serve Nautobot at `/nautobot`, you must set `FORCE_SCRIPT_NAME = "/nautobot"`.
```

Comment:
Same question here re: necessity of using FORCE_SCRIPT_NAME if the server is setting SCRIPT_NAME correctly?

Reply:
Once we get rid of circular imports no longer becomes a necessity! :)

Reply:
:-) Would you mind opening a `housekeeping` issue spelling out what all you think would be needed to remove the need for `FORCE_SCRIPT_NAME`?

Reply:
Yeahhhhhh. I don't know what the exact fix is but I imagine it is directly related to all of the `import *` and deliberate object imports smattered everywhere.